stylesheets/_modules.scss

```diff
@@ -11476,15 +11476,15 @@ $contact-modal-padding: 18px;
 }
 
 /* Spec: container < 437px */
-@media (min-width: 0px) and (max-width: 799px) {
+@media (min-width: 0px) and (max-width: 834px) {
   .module-message {
     // Add 2px for 1px border
     max-width: 302px;
   }
 }
 
 /* Spec: container > 438px and container < 593px */
```

Comment:
Do these comments need to change?

Reply:
Perhaps they do have to change. I don't understand what they're referring to. If that `Spec: container < 437 px` refers to the `module-timeline__message-container`, then they're not up to date.
Should I update them?

Reply:
Yes, I think so.

Reply:
I'm thinking it's this container as mentioned before 
![image](https://user-images.githubusercontent.com/28270981/114536263-30cda680-9c51-11eb-8e8b-948d9a424680.png)

Before my change the values in the comments didn't match that container. I've updated them now.